mergify_engine/tests/unit/test_clients.py

```diff
@@ -28,44 +28,78 @@
 from mergify_engine.clients import http
 
 
+gh_fake_response = {
+    "id": 12345,
+    "target_type": "User",
+    "permissions": {
+        "checks": "write",
+        "contents": "write",
+        "pull_requests": "write",
+    },
+    "account": {"login": "testing", "id": 12345},
+}
+
+
+async def _httpserver_connect(
```

Comment:
The name is too generic,  something like `_do_test_client_installation_token()`.